gemrb/core/DialogHandler.cpp

```diff
@@ -163,8 +158,20 @@ bool DialogHandler::InitDialog(Scriptable* spk, Scriptable* tgt, const ResRef& d
 		Log(DEBUG, "DialogHandler", "Could not find a proper state");
 		return false;
 	}
+	
+	Actor* dialogPC = GetSpeaker();
+	if (dialogPC && dialogPC->InParty) {
+		core->GetDictionary()->SetAt("DIALOG_PC", dialogPC->InParty);
+	} else if (tgt->Type == ST_ACTOR) {
+		core->GetDictionary()->SetAt("DIALOG_PC", static_cast<Actor*>(tgt)->InParty);
+	}
```

Comment:
This is too volatile, the var basically shouldn't be used for anything else but the portrait icon. So a name like DIALOG_STARTER or similar would be more fitting.